target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
After disabling watchdog in Bootmonitor : 

```
[    0.000000] Memory: 122972K/131072K available (4019K kernel code, 168K rwdata, 1172K rodata, 1216K init, 222K bss, 8100K reserved, 0K cma[    0.858966] console [ttyLTQ0] enabledat MMIO 0x1e100c00 (irq = 112, base_baud = 0) is a lantiq,ascHat, Inc.
[    0.866215] bootconsole [early0] disabled
[    0.876047] lantiq nor flash device: 02000000 at 10000000
[    0.881766] ltq-nor 10000000.nor-boot: probing failed
[    0.994611] libphy: lantiq,xrx200-mdio: probed
[    1.069040] eth0: attached PHY [Lantiq XWAY VR9 GPHY 11G v1.4] (phy_addr=0:13, irq=-1)
[    1.141037] eth0: attached PHY [Lantiq XWAY VR9 GPHY 11G v1.4] (phy_addr=0:11, irq=-1)
[    1.213070] eth0: attached PHY [Lantiq XWAY PEF7071] (phy_addr=0:00, irq=-1)
[    1.285052] eth0: attached PHY [Lantiq XWAY PEF7071] (phy_addr=0:01, irq=-1)
[    1.357053] eth0: attached PHY [Lantiq XWAY PEF7071] (phy_addr=0:02, irq=-1)
[    1.366127] wdt 1f8803f0.watchdog: Init done
[    1.375076] NET: Registered protocol family 10
[    1.381277] NET: Registered protocol family 17
[    1.385798] bridge: automatic filtering via arp/ip/ip6tables has been deprecated. Update your scripts to load br_netfilter if you need this.
[    1.398351] 8021q: 802.1Q VLAN Support v1.8
[    1.408270] VFS: Cannot open root device "(null)" or unknown-block(0,0): error -6
[    1.414372] Please append a correct "root=" boot option; here are the available partitions:
[    1.422724] Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0)
[    1.432961] ---[ end Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0)
[   31.280019] random: nonblocking pool is initialized
```
To make it work without u-boot, I have to : 

- Disable watchdog
- Find why the nor flash doesn't work (I think because  endianess issue)